malduck/procmem/procmem.py

```diff
@@ -463,62 +538,78 @@ def patchv(self, addr, buf):
         region = self.addr_region(addr)
         # Boundary check
         if region is None or region.end < (addr + len(buf)):
-            raise ValueError("Cross-region patching is not supported")
+            raise ValueError("Address is out of single region")
```

Comment:
Maybe something like `Patched range spans across multiple regions`?

Reply:
Actually it's not true, because range can be out of any regions.